psydac/linalg/null.py

```diff
@@ -65,6 +65,9 @@ def dot( self, v, out=None ):
 
         return self.codomain.zeros()
 
+    def transpose(self, conjugate=False):
+        return NotImplementedError()
```

Comment:
You should `raise` the exception instead of returning it:
```suggestion
    def transpose(self, conjugate=False):
        raise NotImplementedError()
```